src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,34 @@
+
+from __future__ import absolute_import, division
+
+from celery.utils.log import get_task_logger
+from django.utils import timezone
+
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+from sentry.utils.cache import Lock, UnableToGetLock
+
+logger = get_task_logger(__name__)
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    lock_key = 'scheduler:process'
+    try:
+        with Lock(lock_key, nowait=True, timeout=60):
+            queryset = list(ScheduledJob.objects.filter(
+                date_scheduled__lte=timezone.now(),
+            )[:100])
+
+            for job in queryset:
+                logger.info('Sending scheduled job %s with payload %r',
+                            job.name, job.payload)
+                app.send_task(job.name, kwargs=job.payload)
```

Comment:
Instead of jobs.payload, I'd say we should support both kwargs and args, if the idea is that this is generic for any queued job. Or payload can just contain `{"kwargs": {}, "args": []}`, and we do:  `app.send_task(job.name, **job.payload)` here.